account_billing/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    'name': 'Billing Process',
+    'summary': 'Group invoice as billing before payment',
+    'version': '12.0.1.0.0',
```

Comment:
If Added New Module then the version should be `12.0.0.0.0`

Reply:
Thank you for your reviewed. But I think 1 is mean first version in odoo12.

Reply:
Than It's Keep it `12.0.1.0.0`